EmbyReports/EmbyReports.csproj

```diff
@@ -1,19 +1,21 @@
 <Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;</TargetFrameworks>
+        <AssemblyVersion>1.0.18.0</AssemblyVersion>
```

Comment:
What's up with the lower version?
xml indentation is 2 spaces.

Reply:
Last I checked was with the big formatting fix PR where the xml initially had 4 indents, at the time I wrote this I assumed it was still like that but after poking around in jellyfin I have noticed you are going for 2 spaces. Will fix this